formats/shared/definitions/multi_variant_pages.jsonnet

```diff
@@ -0,0 +1,28 @@
+{
+  transaction_variants: {
+    type: "array",
+    items: {
+      type: "object",
+      additionalProperties: false,
+      required: [
+        "title",
+        "slug",
+        "transaction_start_link",
+      ],
+      properties: {
+        title: {
+          type: "string",
+        },
+        slug: {
+          type: "string",
+          format: "uri",
+        },
+        transaction_start_link: {
```

Comment:
I wonder if we could pull this out into a definition since it's referenced here and in the original `transaction.jsonnet` schema file.

Reply:
I've moved the whole lot back into the `transaction` file since it's only used there.